src/test/java/no/unit/nva/model/instancetypes/NonPeerReviewedTest.java

```diff
@@ -0,0 +1,29 @@
+package no.unit.nva.model.instancetypes;
+
+import no.unit.nva.model.pages.MonographPages;
+import no.unit.nva.model.pages.Range;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import java.rmi.UnexpectedException;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class NonPeerReviewedTest {
+
+    @SuppressWarnings("ConstantConditions")
+    @Test
+    public void nonPeerReviewedMonographPagesSettingPeerReviewedTrueThrowsUnexpectedException() {
+        NonPeerReviewed<MonographPages> nonPeerReviewed = new NonPeerReviewedMonograph();
+        Executable executable = () -> nonPeerReviewed.setPeerReviewed(true);
+        assertThrows(UnexpectedException.class, executable);
```

Comment:
I hate to do that but I was curious what the UnexpectedException was :) I suggest we can leave it for now, but maybe IllegalArgumentException would be more appropriate?

From the documentation of UnexpectedException:
```
/**
 * An <code>UnexpectedException</code> is thrown if the client of a
 * remote method call receives, as a result of the call, a checked
 * exception that is not among the checked exception types declared in the
 * <code>throws</code> clause of the method in the remote interface.
 *
 * @author  Roger Riggs
 * @since   1.1
 */
```

Reply:
We can create a task for this, or just use the IllegalArgumentException if it does not cause many problems.